omod/src/main/java/org/openmrs/module/fhirproxy/web/controller/DelegatingController.java

```diff
@@ -9,39 +9,39 @@
  */
 package org.openmrs.module.fhirproxy.web.controller;
 
-import static java.nio.charset.StandardCharsets.UTF_8;
-import static java.util.Base64.getEncoder;
-import static org.springframework.http.HttpMethod.GET;
-
-import java.io.IOException;
-
-import javax.servlet.http.HttpServletRequest;
-
+import lombok.extern.slf4j.Slf4j;
 import org.openmrs.module.fhirproxy.Config;
 import org.openmrs.module.fhirproxy.FhirProxyUtils;
 import org.openmrs.module.fhirproxy.web.ProxyWebConstants;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import org.springframework.http.HttpEntity;
 import org.springframework.http.HttpHeaders;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.HttpClientErrorException;
 import org.springframework.web.client.RestTemplate;
 import org.springframework.web.util.UriComponentsBuilder;
 
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static java.util.Base64.getEncoder;
+import static org.springframework.http.HttpMethod.GET;
+
 /**
  * Provides a proxy mechanism for all GET requests for ChargeItemDefinition and InventoryItem FHIR
  * resources by delegating to a configured external API to process the request.
  */
+@Slf4j
```

Comment:
I actually don't like using lombok to generate loggers, because they are anti pattern, as in they are in lower case.